### PR TITLE
Add Wisp Super Staff Bros

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

**Comment by panpawn** [comment]
hey so this isn't how we make pull requests

first of all, squash those fkin commits
second of all, come up with an appropriate commit/pr name

please read CONTRIBUTING.md for future reference, thanks

File: chat-plugins/wssb.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		if (target === 'credits') return this.parse('/wssbcredits');
+		let targetData = getMonData(toId(target));
+		if (!targetData) return this.errorReply("The staffmon '" + toId(target) + "' could not be found.");
+		return this.sendReplyBox(targetData);
+	},
+
+	ssbhelp: 'wssbhelp',
+	wssbhelp: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		return this.sendReplyBox("/Wssb [staff member\'s name] - displays data for a staffmon\'s movepool, custom move, and custom ability.");
```
Comment (panpawn) [inline chat-plugins/wssb.js:36]: this isn't how this help command should be done... just use the standard help command system

`wssbhelp: ["Usage: /wssb [staff member's name] - displays data for a staffmon's movepool, custom move, and custom ability."],`

File: chat-plugins/wssb.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		}
+	});
+	return returnData;
+}
+
+exports.commands = {
+	ssb: 'wssb',
+	wssb: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+		if (!target || target === 'help') return this.parse('/help Wssb');
```
Comment (panpawn) [inline chat-plugins/wssb.js]: why is that an uppercase `W`?

File: config/formats.js [hunk 3]
```diff
@@ -797,7 +797,649 @@ exports.Formats = [
...
+		team: 'randomWispStaffBros',
+		ruleset: ['Pokemon', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
+		onBegin: function () {
+			// This seasonal gets a bit from Super Smash Bros., that's where the initial message comes from.
+			this.add('message', "3");
+			this.add('message', "2");
+			this.add('message', "1");
+			this.add('message', "GET READY!");
+			this.add('message', "GOOOO!!!");
+			this.add('raw|<div class="infobox"><center><b>Wisp Super Staff Bros Credits:</b></center><b>%Emg TechP Volco</b> - Programming, Organization, Testing<br /><b>@LotusChamp Torpid</b> - Programming, Organization, Testing<br /><b>Wisp Staff Members</b> - Cooperation and understanding about the project (and some provided aid)');
```
Comment (panpawn) [inline config/formats.js:813]: why?
... no...?

comment credits should be enough, and the credits command is even pushing it...

**Comment by panpawn** [comment]
ok
do
not
make
more
prs
for
this
again

learn
to
use
git

**Comment by Volco** [comment]
fml i fucked up and accidently did all this while trying to fix up my github on widnows

**Comment by panpawn** [comment]
This is being taken over by https://github.com/jd4564/Pokemon-Showdown/pull/124

**Pull request closed without merging**